vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -46,13 +56,31 @@
     - name: Install docker
       block:
         - name: Add Docker’s GPG key
-          apt_key:
+          get_url:
             url: https://download.docker.com/linux/ubuntu/gpg
-            state: present
+            dest: /etc/apt/keyrings/docker.asc
+            mode: '0644'
+            force: true
+          
+        - name: Dearmor GPG key
+          shell: | 
+            cat /etc/apt/keyrings/docker.asc | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg --yes
+                
+        # TODO: (Sunnatillo) Remove this task after fully removing apt-key
+        - name: Remove Docker old repository(without gpg key file location) 
```

Comment:
Nit: add space between `repository` and `(` for readability.

Reply:
done